signac/contrib/project.py

```diff
@@ -2271,7 +2271,7 @@ def next(self):
 
         """
         warnings.warn("Calling next() directly on a JobsCursor is deprecated!"
-                      "Use iter(..).next() instead.", DeprecationWarning)
+                      "Use next(iter(JobsCursor)) instead.", DeprecationWarning)
```

Comment:
That's not quite right actually, since you would not want to call it on the `JobsCursor` class, but on the instance. So I suggest to keep using the ellipsis:
```suggestion
                      "Use next(iter(..)) instead.", DeprecationWarning)
```